src/es/components/molecules/multiLevelNavigation/MultiLevelNavigation.js

```diff
@@ -190,6 +189,15 @@ export default class MultiLevelNavigation extends Mutation() {
   }
 
   connectedCallback () {
+    const preventDefaultInputSearch = this.root.querySelectorAll("a-input[prevent-default-input-search='true']")
+    if (preventDefaultInputSearch.length > 0) {
+      const noScroll = () => {
+          window.scroll(0,0)
+      }
+      preventDefaultInputSearch.forEach(input => {
+        input.addEventListener('blur', noScroll)
```

Comment:
the event listeners should be removed at disconnectedCallback. Events would be attached more than once, in case the component would connect multiple times.